DSharpPlus/Clients/DiscordClient.Dispatch.cs

```diff
@@ -322,6 +322,38 @@ internal async Task HandleDispatchAsync(GatewayPayload payload)
 
                 #endregion
 
+                #region Interaction/Integration/Application
+
+                case "interaction_create":
+                    mbr = dat["member"].ToObject<TransportMember>();
+                    cid = (ulong)dat["channel_id"];
+                    gid = (ulong)dat["guild_id"];
+                    await OnInteractionCreateAsync(gid, cid, mbr, dat.ToObject<DiscordInteraction>()).ConfigureAwait(false);
+                    break;
+                
+                // The following are not documented and thus ignored for the time being.
+                // Please update these if they are documented :)
```

Comment:
The application command create, update and delete are indeed documented [here](https://discord.com/developers/docs/topics/gateway#commands), the docs website just hasn't been updated to show them on the events table yet

Reply:
Did not notice those, thank you for telling me. I will add those in along with the relevant objects :)